assemblies/assembly-about-rhdh.adoc

```diff
@@ -0,0 +1,21 @@
+ifdef::context[:parent-context-of-about-rhdh: {context}]
+
+:_mod-docs-content-type: ASSEMBLY
+
+ifndef::context[]
+[id="about-rhdh"]
+endif::[]
+ifdef::context[]
+[id="about-rhdh_{context}"]
+endif::[]
+= About {product}
+
+:context: about-rhdh
+
+{product} is a fully supported, open developer platform that reduces friction and frustration of developers while boosting productivity. 
+
+This platform is driven by a centralized software catalog, providing efficiency to your microservices and infrastructure. 
+
+Use {product} to simplify decision-making through a selection of internally approved tools, programming languages, and developer resources within a self-managed portal. 
+
+include::modules/discover/con-benefits-of-rhdh.adoc[]
```

Comment:
To avoid confusion, do not add the same content in two separate pull request. This procedure that is already present in https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/663

Instead, add the supported platforms content in a new module and add the module:

```suggestion
include::modules/discover/con-supported-platforms.adoc[]
```